store/store.go

```diff
@@ -0,0 +1,16 @@
+package store
+
+import "github.com/gopheracademy/congo"
+
+// Interface is a super interface of all substores.
+type Interface interface {
+	UserStore
+}
+
+// UserStore represents the interface for user storage.
+type UserStore interface {
+	User(id int) (*congo.User, error)
+	UserByEmail(email string) (*congo.User, error)
+	CreateUser(u *congo.User) error
+	DeleteUser(id uint64) error
+}
```

Comment:
This is the interface that our Bolt implementation, `*bolt.UserStore`, implements. Other types of backends could implement this same interface in the future. The rest of our code shouldn't care how the data is stored behind the scenes.